### DISCUSSION TITLE
Is there a way to enable PKHaX mode on Linux?

### DISCUSSION THREAD

**Discussion by reisscake** [kwsch/PKHeX#3540] [Q&A] [answered]

**Comment by kwsch** [comment, marked as answer]
There are three ways:
1. Rename the executable from PKHeX to PKHaX (case insensitive)
2. Launch the program with -hax command line argument
3. Open the settings and change the startup setting to force hax on startup.

**Comment by ijuintekka** [comment]
Renaming the executable won't work with (current) mono, as Environment.ProcessPath returns the path to mono-sgen instead of the executable. Works fine on wine though.

**Reply by FrostKnight** [reply]
Yeah, I actually just tried using wine on linux and it worked for me. I used wine-staging though.
